benchopt/datasets/simulated.py

```diff
@@ -84,6 +89,11 @@ def make_correlated_data(
     else:
         X = rng.randn(n_samples, n_features)
 
+    if X_density != 1:
```

Comment:
import scipy sparse here or just after the docstring. We do not want to have to import scipy sparse when importing benchopt. Typically  it slows down a lot the autocomplete